config/samples/nim/llm/multi-node/multi-node-nimservice-infiniband.yaml

```diff
@@ -0,0 +1,93 @@
+
+apiVersion: apps.nvidia.com/v1alpha1
+kind: NIMService
+metadata:
+  name: deepseek-r1
+spec:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: example-ipoibnetwork   # configured by NVIDIA Network Operator
+  env:
+  - name: NCCL_DEBUG
+    value: "WARN"
+  - name: "NCCL_NET"
+    value: "Socket"
+  - name: NIM_USE_SGLANG
+    value: "1"
+  - name: NIM_MULTI_NODE
+    value: "1"
+  - name: NIM_TENSOR_PARALLEL_SIZE
+    value: '8'
+  - name: NIM_PIPELINE_PARALLEL_SIZE
+    value: '2'
+  - name: NGC_HOME
+    value: /model-store/ngc/hub
+  - name: HF_HOME
+    value: /model-store/huggingface/hub
+  - name: NUMBA_CACHE_DIR
+    value: /tmp/numba
+  - name: OUTLINES_CACHE_DIR
+    value: /tmp/outlines
+  - name: UCX_TLS
+    value: ib,tcp,shm
+  - name: UCC_TLS
+    value: ucp
+  - name: UCC_CONFIG_FILE
+    value: " "
+  - name: GLOO_SOCKET_IFNAME
+    value: eth0
+  - name: NCCL_SOCKET_IFNAME
+    value: net1
```

Comment:
can this be configured via DRA ResourceClaim; If yes can you please add an example? thanks